src/components/ha-labeled-slider.js

```diff
@@ -24,19 +24,20 @@ class HaLabeledSlider extends PolymerElement {
         opacity: var(--dark-secondary-opacity);
       }
 
-      .slider-container {
-        margin-left: 24px;
-      }
-
       ha-paper-slider {
         background-image: var(--ha-slider-background);
       }
     </style>
 
     <div class="title">[[caption]]</div>
-    <iron-icon icon="[[icon]]"></iron-icon>
+    <template is="dom-if" if="[[extra]]">
+      <div class="extra-container">
+        <slot></slot>
```

Comment:
let's give the slot a name. I also don't think that we need a `dom-if` around it. It won't be rendered if it's not there.